src/Farmer/Builders/Builders.Functions.fs

```diff
@@ -323,6 +323,12 @@ type FunctionsConfig =
                 { site with AppSettings = None; ConnectionStrings = None }
                 for (_, slot) in this.CommonWebConfig.Slots |> Map.toSeq do
                     slot.ToSite site
+                    
+            if this.CommonWebConfig.SlotSettingNames <> Set.empty then
+                {
```

Comment:
Formatting - I actually do prefer this style but the rest of Farmer uses the more common record creation style, could you follow that? Also `;` unnecessary and can be removed.